Apps/LekaApp/Sources/Views/Categories/CategoryResourcesDeepDiveView.swift

```diff
@@ -64,7 +64,7 @@ struct CategoryResourcesDeepDiveView: View {
 
     // MARK: Private
 
-    private let category: CategoryResources = ContentKit.deepDiveResources
+    private let category: CategoryResources = ContentKit.allResources.first(where: { $0.id == "D222543E98A8419DB44A8FE2D3B9BD56" })!
```

Comment:
it feels a bit strange to use the uuid like this.
maybe for the resources we should take a different approach and use the "name" instead.